baseapp/baseapp.go

```diff
@@ -970,7 +970,7 @@ func (app *BaseApp) runTx(ctx sdk.Context, mode runTxMode, txBytes []byte) (gInf
 		// Dont need to validate in checkTx mode
 		if ctx.MsgValidator() != nil && mode == runTxModeDeliver {
 			storeAccessOpEvents := msCache.GetEvents()
-			accessOps, _ := app.anteDepGenerator([]acltypes.AccessOperation{}, tx)
+			accessOps := ctx.TxMsgAccessOps()[acltypes.ANTE_MSG_INDEX]
```

Comment:
cc @BrandonWeng is there a specific reason you have previously called the dependency generator here? I assume it was because you may not have been aware that the `-1` key was reserved for the access ops generated for the ante handler ahead of time? but just want to make sure there wasnt some other reason

Reply:
Teah that was most likely the case